From 85471b5821c86b88221b40ee110a8588f0dc39c1 Mon Sep 17 00:00:00 2001
From: Nicolin Chen <Guangyu.Chen@freescale.com>
Date: Wed, 9 Apr 2014 17:12:12 +0800
Subject: [PATCH] ENGR00307635-6 ASoC: fsl: Enable SAI for imx-sgtl5000 and
 imx-wm8962

Since we are able to link SAI and sgtl5000 and wm8962, we should update
the Kconfig to make it build-in if enabling their machine drivers.

Acked-by: Wang Shengjiu <b02247@freescale.com>
Signed-off-by: Nicolin Chen <Guangyu.Chen@freescale.com>
(cherry picked from commit 5cf1d7bcc43ff45b60b90981845df03f20aede87)
---
 sound/soc/fsl/Kconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/sound/soc/fsl/Kconfig b/sound/soc/fsl/Kconfig
index faf9797..d48aaa1 100644
--- a/sound/soc/fsl/Kconfig
+++ b/sound/soc/fsl/Kconfig
@@ -258,6 +258,7 @@ config SND_SOC_IMX_WM8962
 	select SND_SOC_WM8962
 	select SND_SOC_IMX_PCM_DMA
 	select SND_SOC_IMX_AUDMUX
+	select SND_SOC_FSL_SAI
 	select SND_SOC_FSL_SSI
 	select SND_KCTL_JACK
 	help
@@ -277,6 +278,7 @@ config SND_SOC_IMX_SGTL5000
 	select SND_SOC_SGTL5000
 	select SND_SOC_IMX_PCM_DMA
 	select SND_SOC_IMX_AUDMUX
+	select SND_SOC_FSL_SAI
 	select SND_SOC_FSL_SSI
 	help
 	  Say Y if you want to add support for SoC audio on an i.MX board with
-- 
1.8.0

